Makefile

```diff
@@ -4,15 +4,19 @@ GIT_VERSION ?= $(shell git describe --tags --always --dirty)
 
 LDFLAGS=-buildid= -X github.com/gittuf/gittuf/internal/version.gitVersion=$(GIT_VERSION)
 
-.PHONY : build test install
+.PHONY : build test install fmt
 
 default : install
 
 build : test
-	CGO_ENABLED=0 go build -trimpath -ldflags "$(LDFLAGS)" .
+	mkdir -p ./dist
```

Comment:
You don't need mkdir, go build will create the directory if needed. Also note that this may again not be portable, I'm not aware if mkdir is supported as an alias on Windows.

Reply:
I took out the mkdir